src/components/organisms/Chatbox/Chatbox.tsx

```diff
@@ -1,27 +1,31 @@
 import React, { useState, useEffect, useMemo, useRef } from "react";
-import UserProfileModal from "components/organisms/UserProfileModal";
+import { useFirestoreConnect } from "react-redux-firebase";
```

Comment:
Should we be importing this from our 'wrapped' version rather than directly from the library?

Reply:
I didn't import it myself. I have only changed the import order. It would be nice to have a commit to remove `useFirestoreConnect` across our codebase. Didn't do it in this PR as it is unrelated to it. Though, I think we should create a task for that.

Reply:
https://github.com/sparkletown/sparkle/pull/801 is the WIP PR that is implementing this + eslint banning the direct usage across the codebase